components/src/test/java/org/folio/rest/workflow/model/ConditionTest.java

```diff
@@ -107,4 +114,65 @@ void setAnswerWorksTest() {
     assertEquals(VALUE, getField(condition, "answer"));
   }
 
+  @ParameterizedTest
+  @MethodSource("providePrePersistFor")
+  void prePersistWorksTest(Map<String, Object> initial, Map<String, Object> expected) {
+    initial.forEach((String attribute, Object value) -> {
+      setField(condition, attribute, value);
+    });
+
+    condition.prePersist();
+
+    expected.forEach((String attribute, Object value) -> {
+      assertEquals(value, getField(condition, attribute));
+    });
+  }
+
+  /**
+   * Helper function for parameterized tests for the prePersist function.
+   *
+   * @return
+   *   The arguments array stream with the stream columns as:
+   *     - Arguments initial The initial values.
+   *     - Arguments expect The expected values.
+   */
+  private static Stream<Arguments> providePrePersistFor() {
+
+    return Stream.of(
```

Comment:
These tests need to use `VALUE` for the string tests to test that `VALUE` gets assigned.
Sorry, I forgot to check for that and request that in the previous PR review.

Here is an example:
```java
    return Stream.of(
      Arguments.of(
        helperFieldMap(null,  null),
        helperFieldMap("",    "")
      ),
      Arguments.of(
        helperFieldMap(VALUE, null),
        helperFieldMap(VALUE, "")
      ),
      Arguments.of(
        helperFieldMap(null,  VALUE),
        helperFieldMap("",    VALUE)
      ),
      Arguments.of(
        helperFieldMap(VALUE, VALUE),
        helperFieldMap(VALUE, VALUE)
      )
    );
```

Reply:
VALUE to be used for string tests  - suggestion incorporated for all other string test cases as well.